src/Discord.Net/Models/Emotes/Emote.cs

```diff
@@ -0,0 +1,18 @@
+namespace Discord
+{
+    // placeholder for user-constructed guild emotes
+    // TODO: naming - should be called GuildEmote? but does not impl IGuildEmote, so maybe not...
+    internal class Emote : IEmote
+    {
+        public Emote(ulong id, string name)
```

Comment:
I'd swap the parameter order here to keep consistent with the `<:name:id>` format.